packages/datadog_flutter_plugin/android/src/test/kotlin/com/datadoghq/flutter/DatadogSdkPluginTest.kt

```diff
@@ -157,6 +162,87 @@ class DatadogSdkPluginTest {
         verify(mockResult).success(null)
     }
 
+    @Test
+    fun `M not issue warning W initialize called with same configuration`(
+        forge: Forge
+    ) {
+        // GIVEN
+        mockkStatic(Log::class)
```

Comment:
personally I'm not big fan of static mocks, because to mock static bytecode modification is needed (this is what `mockk` is doing under the hood, probably).

maybe we can do non-static wrapper around `Log`?

Anyway, is not a blocker for me, just a personal opinion :)